### ISSUE TITLE
Dropdown bug?

### ISSUE THREAD

**Issue by nikkii94** [wakirin/Litepicker#54]
**Describe the bug**
I tried to select year and month from the dropdowns, but most of the options shown as disabled.

**To Reproduce**
My settings are these:
```
minDate: "2016-01-01"
maxDate: "2020-12-31"
dropdowns: {
    minYear: 2016
    maxYear: 2020
    months: true
    years: true
}
```
It only allows to select 2020 and december.

**jsfiddle.net / codepen.io / etc**
https://jsfiddle.net/jq38cw7r/5/

**Additional context**
Maybe I am wrong, but shouldn't it be isAfter for the second condition?
https://github.com/wakirin/Litepicker/blob/master/src/calendar.ts#L164

Thank you for this great library!

**Comment by wakirin** [comment]
Nice catch!
Fixed in v1.0.35

**Comment by ikabod** [comment]
It is broken again.

If you look at the fiddle, 2020 is set for maxYear, but the calendar shows 2022 in the dropdown.
